osm2lanes/src/transform/tags_to_lanes/modes/non_motorized.rs

```diff
@@ -1,22 +1,22 @@
 use crate::locale::{DrivingSide, Locale};
 use crate::road::{Designated, Lane, Road};
-use crate::tag::{Tags, HIGHWAY};
+use crate::tag::{HighwayType, Tags, HIGHWAY};
 use crate::transform::tags_to_lanes::{RoadBuilder, TagsToLanesMsg};
 use crate::transform::{RoadError, RoadFromTags, RoadWarnings};
 
 impl Lane {
     fn shoulder(locale: &Locale) -> Self {
         Self::Shoulder {
             // TODO: width not just motor
-            width: Some(locale.travel_width(&Designated::Motor)),
+            width: Some(locale.travel_width(&Designated::Motor, HighwayType::UnknownRoad)),
```

Comment:
Same as other comment -- are these basically TODOs for more refactoring+plumbing, or do we ever want to pass UnknownRoad?

Reply:
this definitely needs a TODO, or I will rewrite it to be better...

Reply:
actually, this whole file looks like a mess, https://github.com/a-b-street/osm2lanes/issues/157